lib/oracledb.js

```diff
@@ -22,13 +22,33 @@ var oracledbInst;
 var Lob = require('./lob.js').Lob;
 var pool = require('./pool.js');
 var connection = require('./connection.js');
+var path;
+var existsSync;
 
 try {
   oracledbCLib =  require('../build/Release/oracledb');
 } catch (err) {
   if (err.code === 'MODULE_NOT_FOUND') {
     oracledbCLib = require('../build/Debug/oracledb');
   } else {
+    
+    path = require('path');
+    existsSync = require('fs').existsSync || require('path').existsSync;
+    
+    process.env.PATH.split(path.delimiter).forEach(function(dir) {
+      if(existsSync(path.join(dir, 'oci.dll'))) {
+		  console.log();
+		  console.log('**** Found Oracle Instant Client in PATH in the following direcory:' + dir);
```

Comment:
For non-Windows I think you'd be OK checking for 'libclntsh.*'  Harder to portably check the link has been created.